sql/telemetry_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -39,17 +39,16 @@ WITH
   FROM
     `moz-fx-data-derived-datasets.telemetry.fxa_content_auth_oauth_events_v1`
   WHERE
-    ((event_type IN ('fxa_login - complete', 'fxa_reg - complete'))
+    ((event_type IN ('fxa_login - complete', 'fxa_reg - complete') AND service IS NOT NULL)
```

Comment:
The where clause doesn't have access to the redefined `service` field, so this is using the raw value of `service` and will still filter out the sync events we want to include. I'm going to push another commit to factor the redefinition of `service` to a base CTE, and then we can use this filter here.